Content.Server/Wires/WiresSystem.cs

```diff
@@ -33,6 +32,8 @@ public sealed class WiresSystem : EntitySystem
     [Dependency] private readonly HandsSystem _handsSystem = default!;
     [Dependency] private readonly DoAfterSystem _doAfter = default!;
 
+    private IRobustRandom _random = new RobustRandom();
+
```

Comment:
This one's a bit sussy, I'd just make a new System.Random for now.

Reply:
the thing with this is that IRobustRandom's interface has a list shuffler in it, so i have to use it unless i want to change up how the wire randomization works (which is probably just gonna be me copying and pasting that interface function tbh)